Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -310,6 +440,16 @@ namespace pcpp
 		}
 #endif
 
+		if (config.timestampProvider != PcapLiveDevice::TimestampProvider::Host)
+		{
+			setTimestampProvider(pcap, config.timestampProvider);
+		}
+
+		if (config.timestampPrecision != PcapLiveDevice::TimestampPrecision::Nanoseconds)
```

Comment:
Shouldn't this be `if (config.timestampPrecision != PcapLiveDevice::TimestampPrecision::Microseconds)`?

Because microsecond precision is the default...

Reply:
Fixed in this commit: https://github.com/seladb/PcapPlusPlus/commit/5251602c9b95ca0229c2a78fbda199c1816482d0